app-admin/logstash-bin/logstash-bin-2.3.3.ebuild

```diff
@@ -11,12 +11,13 @@ MY_P="${MY_PN}-${PV}"
 
 DESCRIPTION="Tool for managing events and logs"
 HOMEPAGE="https://www.elastic.co/products/logstash"
-SRC_URI="https://download.elastic.co/${MY_PN}/${MY_PN}/${MY_P}.tar.gz"
+SRC_URI="standard? ( https://download.elastic.co/${MY_PN}/${MY_PN}/${MY_P}.tar.gz )
+	all-plugins? ( https://download.elastic.co/${MY_PN}/${MY_PN}/${MY_PN}-all-plugins-${PV}.tar.gz )"
 
 LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE=""
+IUSE="+standard all-plugins"
```

Comment:
This does not behave exactly the way you intended it, assuming you want: "if the user specifies nothing, pick standard". When the user specifies USE=all-plugins, this will require explicit USE=-standard. i.e.

```
$ USE="standard" ebuild logstash-bin-2.3.3.ebuild pretend
# Success

$ USE="all-plugins" ebuild logstash-bin-2.3.3.ebuild pretend
  The following REQUIRED_USE flag constraints are unsatisfied:
    ?? ( standard all-plugins )

$ USE="all-plugins -standard" ebuild logstash-bin-2.3.3.ebuild pretend
# Success
```

Sorry for not realizing this earlier. In this case, you can accomplish that behavior by removing the "standard" USE flag altogether and using negation on the "all-plugins" flag like so:

```
SRC_URI="!all-plugins? ( https://download.elastic.co/${MY_PN}/${MY_PN}/${MY_P}.tar.gz )                                                 
    all-plugins? ( https://download.elastic.co/${MY_PN}/${MY_PN}/${MY_PN}-all-plugins-${PV}.tar.gz )"
```

Since logstash-bin-2.3.1 already had the USE flags, it doesn't break anything. In the interest of time, I will merge this. It may be worth considering this as a future enhancement though.